.circleci/config.yml

```diff
@@ -362,6 +367,26 @@ workflows:
           requires:
             - prep-build-ts-migration-dashboard
 
+  locales_only:
+    when:
+      matches:
+        pattern: /^l10n_crowdin_action$/
+        value: << pipeline.git.branch >>
+    jobs:
+      - prep-deps
+      - get-changed-files-with-git-diff:
+          <<: *exclude_develop_master_rc
+          requires:
+            - prep-deps
+      - validate-locales-only:
+          requires:
+            - get-changed-files-with-git-diff
+      - test-lint:
```

Comment:
This workflow is missing an `all-tests-pass` step, which is what we use as a PR status check to block merging.